azure/edgedb-flexibleserver.json

```diff
@@ -0,0 +1,217 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "minLength": 4,
+            "maxLength": 63,
+            "metadata": {
+                "description": "Unique name for the deployment, used for the Postgres server and EdgeDB container."
+            }
+        },
+        "postgresAdminUsername": {
+            "type": "string",
+            "minLength": 4,
+            "maxLength": 128,
+            "metadata": {
+                "description": "Administrator username for Postgres."
+            }
+        },
+        "postgresAdminPassword": {
```

Comment:
Let's use the same password for both Postgres and EdgeDB to reduce the number of knobs.